tests/test_models/test_nn/deepar/test_loss.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import torch
+from pytorch_lightning import seed_everything
+
+from etna.models.nn.deepar_new.loss import GaussianLoss
+from etna.models.nn.deepar_new.loss import NegativeBinomialLoss
+
+
+@pytest.mark.parametrize(
+    "loss,inputs,n_samples,expected_sample",
+    [
+        (GaussianLoss(), (torch.tensor([0.0]), torch.tensor([1.0]), torch.tensor([1.0])), 2, torch.tensor([1.5410])),
```

Comment:
How it is possible to check that it sample some expected vector in random sampling case?

Reply:
I guess there should be 2 tests:
- Deterministic -- for theoretical mean
- Random -- for random sampling(there you can check that emperical mean/std of samples if close to theoretical)